src/api-reference/request/v4.endpoints.request-resources.markdown

```diff
@@ -208,10 +215,12 @@ GET {datacenter}/travelrequest/v4/requests
+view|query|string|Name of the perimeter defining the Requests to get.<br>Possible values:<br>`ALL`: Get all existing Requests for a user (relevant only for the traveler);<br>`ACTIVE`Get all active Requests. Does not include cancelled Requests. Approved Requests included are aged less than 3 months based on current date and must not be in closed status;<br>`UNSUBMITTED`: Get all the unsubmitted Requests (relevant only for the traveler);<br>`PENDING`: Get all the pending Requests (relevant only for the traveler);<br>`VALIDATED`: Get all the approved Requests for a user (relevant only for the traveler). Closed Requests are included in this view;<br>`APPROVED`: Get all the approved Requests by a user (relevant only for the approver). Closed Requests are included in this view;<br>`CANCELED`: Get all the cancelled Requests for a user (relevant only for the traveler). Cancelled could include close/not closed requests;<br>`CLOSED`: Get all the closed Requests for a user (relevant only for the traveler). Includes canceled then closed Request as well as approved then closed Requests;<br>`SUBMITTED`: Get all the submitted Requests for a user (relevant only for the traveler). Submitted does not include cancelled requests;<br>`TOAPPROVE`: Get all Requests to be approved by the user (relevant only for the approver);<br>`PENDINGEBOOKING`: Approved Requests awaiting Concur Travel booking(s);<br>`PENDINGPROPOSAL`: Get all Requests submitted to a Travel Agency (TMC) step step. The userId of the TMC agent user must be provided in the list of parameters, this TMC agent user must have a default Travel Agency assigned in its profile corresponding to the Travel Agency assigned to the Requests.<br>When connecting with a User Token, additional values are possible: `PROPOSALAPPROVED`, `PROPOSALCANCELED`.<br>If no view value is sent, the default view `ALL` will be used
```

Comment:
A few text edits, also is there a reason to not alphabetize these values? 
"perimeter" should be "parameter"
PENDING: Does pending mean submitted but not yet approved? or not yet final approved (could have some approvals but not all required approvals)? Please clarify.
CANCELED "close/not closed" should be "closed/not closed"
PENDINGPROPOSAL has an extra "step"
PROPOSALAPPROVED and PROPOSALCANCELLED should also have definitions.